packages/labs/src/components/popover/arrow.tsx

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+import * as React from "react";
+import { Arrow } from "react-popper";
+
+import { Classes } from "@blueprintjs/core";
+
+const SVG_SHADOW_PATH = "M8.11 6.302c1.015-.936 1.887-2.922 1.887-4.297v26c0-1.378" +
```

Comment:
_the sketch file has been in `/resources` this whole time_! https://github.com/palantir/blueprint/blob/master/resources/sketch/Core%20Kit.sketch